mlflow/utils/cli_args.py

```diff
@@ -189,7 +189,15 @@ def _create_env_manager_option(help_string, default=None):
     "-w",
     envvar="MLFLOW_WORKERS",
     default=None,
-    help="Number of gunicorn worker processes to handle requests (default: 1).",
+    help="Number of gunicorn worker processes to handle requests (default: 4).",
+)
+
+MODELS_WORKERS = click.option(
```

Comment:
q: what would happen when both WORKERS and MODELS_WORKERS are passed to a command?

Reply:
They shouldn't be used together :)

Reply:
Yup, I wonder if there is any way to prevent them from being used together.